sdk/core/core-lro/package.json

```diff
@@ -92,7 +92,7 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/core-tracing": "^1.0.0",
```

Comment:
core-lro doesn't seem to use core-tracing at all, so I wasn't sure what to do here - should we remove that dependency entirely?

Reply:
yes, core-lro should not use core-tracing. A quick git-blame reveals it was added by mistake in https://github.com/Azure/azure-sdk-for-js/commit/5cd6362fbd47c1b6d2dc10071623a4444ec16a66